test/service/CountriesServiceSpec.scala

```diff
@@ -14,21 +14,20 @@
  * limitations under the License.
  */
 
-package pages
+package service
 
-import org.openqa.selenium.By
-import support.{ITSpec, WireMockResponses}
+package service
 
-class NoVatRepaymentsFoundSpec extends ITSpec with CommonPage {
+import support.ITSpec
 
-  val path = "/vat-repayment-tracker-frontend/show-results/vrn/234567890"
+class CountriesServiceSpec extends ITSpec {
 
-  "user is authorised and no financial data found" in {
-    WireMockResponses.authOk(wireMockBaseUrlAsString = wireMockBaseUrlAsString)
-    WireMockResponses.financialsNotFound
-    goToViaPath(path)
-    webDriver.getTitle shouldBe "Vat Repayment Tracker"
-    probing(_.findElement(By.id("main-message")).getText) shouldBe "No VAT repayments in progress"
-  }
+  val countryService = fakeApplication.injector.instanceOf[CountriesService]
```

Comment:
`fakeApplication` is a method which creates new application. It's better to delegate call to `app`:
see:
```
  /**
   * Implicit method that returns the `Application` instance for the current test.
   */
  implicit final def app: Application = synchronized { privateApp }
```